Bottom: ab7198f69f0aa95dbf145412dd7951df243e17a7
Top:    a0e5d82fe45d95e2f245b0c1513a7db44c5a090e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Update ptxdistrc filename


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
similarity index 100%
rename from docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
rename to docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
